playwright/tests/screenshots/browsers.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { test, expect, type Page } from '@playwright/test';
+import { InscriptionView } from '../pageobjects/InscriptionView';
+import type { Section } from '../pageobjects/Section';
+import { screenshot } from './screenshot-util';
+
+const GENERIC_PID = {
+  SCRIPT: '168F0C6DF682858E-f3',
+  USER_TASK: '168F0C6DF682858E-f5',
+  DATABASE: '18141E75C9CEDD35-f3'
+} as const;
+
+test.describe('Browsers', () => {
+  test('ScriptingBrowser', async ({ page }) => {
+    await openSection(page, GENERIC_PID.SCRIPT, 'Output', 'Code').then(async section => {
+      await section.scriptArea().openBrowsers();
+      const browser = section.scriptArea().browser();
```

Comment:
if you direclty return the browser you can:
```suggestion
      const browser = await section.scriptArea().openBrowsers();
```